example-models/db.js

```diff
@@ -1,8 +1,8 @@
 const Sequelize = require("sequelize");
-const DATABASE_URI = "postgres://localhost/juke";
+const DATABASE_URI = "sqlite://:memory:";
 
 // create the database instance
 module.exports = new Sequelize(DATABASE_URI, {
   logging: false, // set to console.log to see the raw SQL queries
-  native: true // lets Sequelize know we can use pg-native for ~30% more speed
+  // native: true // lets Sequelize know we can use pg-native for ~30% more speed
```

Comment:
Great point. let's just take out for now

Reply:
I agree—using the most portable option is best in my opinion.